js/src/setup-mc/setup-stepper/saved-setup-stepper.js

```diff
@@ -225,7 +226,15 @@ const SavedSetupStepper = ( { savedStep } ) => {
 				{
 					key: stepNameKeyMap.paid_ads,
 					label: __( 'Create a campaign', 'google-listings-and-ads' ),
-					content: <SetupPaidAds />,
+					content: (
+						<CampaignAssetsForm
```

Comment:
Curious if the `SetupPaidAds` component (setup-mc/setup-paid-ads) could include the `CampaignAssetsForm` so this change wouldn't be needed?

Reply:
I didn't think of that 🤣 . Definitely that works!